File: cs50x/week1/meow/meow.c
#include <stdio.h>

void meow(int);

int main(void)
{
    meow(3);
}

void meow(n)
{
    for (int i = 0; i < n; i++)
    {
    printf("meow\n");
    }
}

/*
void meow(void);

int main(void)
{
    for (int i = 0; i < 3; i++)
    {
        meow();
    }
}

void meow(void)
{
    printf("meow\n");
}
*/

File: cs50x/week6/lecture/meow.py
    # What about taking input here?
def main():
    meow(3)

def meow(n):
    for i in range(n):
        print("meow")

main()

    # Need to call main at the bottom
def main():
    for i in range(3):
        meow()

def meow():
    print("meow")

main()

    # Can define a main function like this, like we do in C. 
    # But even then it doesn't work.
# def main():
#     for i in range(3):
#         meow()

# def meow():
#     print("meow")


    # This will not work:
# for i in range(3):
#     meow()

# def meow():
#     print("meow")


# for i in range(3):
#     print("meow")